### DISCUSSION TITLE
More RL and callbacks

### DISCUSSION THREAD

**Discussion by Borda** [PyTorchLightning/pytorch-lightning-bolts#735] [General]
### Added

- Added Soft Actor Critic (SAC) Model (#627)
- Added `EMNISTDataModule`, `BinaryEMNISTDataModule`, and `BinaryEMNIST` dataset (#676)
- Added Advantage Actor-Critic (A2C) Model (#598)
- Added Torch ORT Callback (#720)
- Added SparseML Callback (#724)

### Changed

- Changed the default values `pin_memory=False`, `shuffle=False` and `num_workers=16` to `pin_memory=True`, `shuffle=True` and `num_workers=0` of datamodules (#701)
- Supporting deprecated attribute usage (#699)

### Fixed

- Fixed ImageNet val loader to use val transform instead of train transform (#713)
- Fixed the MNIST download giving HTTP 404 with `torchvision>=0.9.1` (#674)
- Removed momentum updating from val step and add separate val queue (#631)
- Fixed moving the queue to GPU when resuming checkpoint for SwAV model (#684)
- Fixed FP16 support with vision GPT model (#694)
- Removing bias from linear model regularisation (#669)
- Fixed CPC module issue (#680)

<hr /><em>This discussion was created from the release <a href='https://github.com/PyTorchLightning/lightning-bolts/releases/tag/0.4.0'>More RL and callbacks</a>.</em>
